lib/utils.js

```diff
@@ -129,6 +129,17 @@ exports.equals = function(actual, expected, ignoredKeys){
   }
 };
 
+exports.contains = function(actual, expected){
+  for (var key in expected) {
+    if (actual[key] !== expected[key]) {
```

Comment:
Used assert.deepEqual instead of exports.equals. Let me know if you think that's missing something?